libcontainer/seccomp/seccomp_linux.go

```diff
@@ -42,7 +42,23 @@ func InitSeccomp(config *configs.Seccomp) (int, error) {
 
 	// Ignore the error since pre-2.4 libseccomp is treated as API level 0.
 	apiLevel, _ := libseccomp.GetAPI()
+	const writeSyscall = "write"
+	writeAllowed := false
 	for _, call := range config.Syscalls {
+		// If write is in the allowed list of syscalls, then we can also notify on the defaultAction.
+		if call.Name == writeSyscall {
+			switch call.Action {
+			case configs.Allow:
+				fallthrough
+			case configs.Trace:
+				fallthrough
+			case configs.Log:
```

Comment:
this could be shorter with `case configs.Allow, configs.Trace, configs.Log:`

Reply:
Yep, improved as suggested :+1: